docs/_sass/td-content.scss

```diff
@@ -16,6 +16,14 @@
     font-weight: 400;
   }
 
+  blockquote {
+    padding: 0.8em 1.2em;
+    margin: 1.2em 0;
+    border-left: 4px solid #dfe2e5;
+    background-color: #f9f9f9;
```

Comment:
Light and dark mode compatible?

Use of `var(--predefined-color)` is likely appropriate.

Reply:
One of these might fit the bill - https://github.com/meshery/meshery/blob/master/docs/_sass/root-variables.scss